dataactcore/models/base.py

```diff
@@ -0,0 +1,4 @@
+from sqlalchemy.ext.declarative import declarative_base
+from dataactcore.utils.timeStampMixin import TimeStampBase
+
+Base = declarative_base(cls=TimeStampBase)
```

Comment:
What is the purpose of this file?  I believe we had different base objects for different database to keep metadata objects separate for sqlalchemy, combining them could be problematic.  If the goal here is just to use this for the files in the validation database, could we give it a more specific name like validationBase.py?